tests/languages/cli/test.js

```diff
@@ -1,6 +1,6 @@
 const { exec, execSync } = require('child_process');
 
-execSync("node index client --endpoint 'https://appwrite.io/v1' --projectId 6013fbefdfa41 --key=35y3h5h345 --selfSigned true", { stdio: 'inherit' });
+execSync("node index client --endpoint 'https://8080-appwrite-appwrite-pdthqc4qqbr.ws-us60.gitpod.io/v1' --projectId console --key=35y3h5h345 --selfSigned true", { stdio: 'inherit' });
```

Comment:
revert back the endpoint details here?

Reply:
Yes, I'll keep this comment open as a reminder to do it 👍